Added class-dump dockerized version to MASTG-TOOL-0043 (#3466)
OWASP/mastg: tools/ios/MASTG-TOOL-0043.md

---

[class-dump](http://stevenygard.com/projects/class-dump/) is a command line utility for examining the Objective-C runtime information stored in Mach-O (Mach object) files. It generates declarations for the classes, categories, and protocols.

> Although the official release of the tool works only in macOS, it is also available in an unofficial [dockerized version](https://github.com/lucacapacci/class-dump-docker), allowing for use across different environments.

Reviewer: ```suggestion

The official release of the tool only works in macOS, but an unofficial [dockerized version](https://github.com/lucacapacci/class-dump-docker) is also available, allowing for use across different environments.
```
